Update README.md (#16)
Xceptance/neodymium-showcase: README.md

# Introduction

> This repository is a showcase project to demonstrate the usage of several features of the [Neodymium Library](https://github.com/Xceptance/neodymium-library). It was created based on the [Neodymium template](https://github.com/Xceptance/neodymium-template).

Reviewer: Is it really important that the Neodymium template was used to create this project?

# Introduction

> This repository is supposed to be used as a showcase project to show the usage of a number of feature for the [Neodymium Library](https://github.com/Xceptance/neodymium-library). It was created based on the [Neodymium template](https://github.com/Xceptance/neodymium-template).

Reviewer: ```suggestion
This repository is a showcase project to demonstrate the usage of several features of the [Neodymium Library](https://github.com/Xceptance/neodymium-library).
```

IMO, it is not important that this project is based on the Neodymium template.

# Content

> Below is a tabel of the content fro this project.

Reviewer: ```suggestion
# Contents
Below is a summary of the contents from this project.
```

## 1. Shadow DOM

> The `showcase.neodymium.tests.shadowdom` package contains the uscases for the shadow DOM automation.

Reviewer: ```suggestion
The `showcase.neodymium.tests.shadowdom` package contains the use cases for the shadow DOM automation.
```

> The `showcase.neodymium.tests.shadowdom` package contains the uscases for the shadow DOM automation.

Reviewer: If we have a wiki page for the shadow DOM or a section in our wiki about this, please place a hyperlink to the wiki page of the feature on "shadow DOM."

#### 1.1 General Shadow DOM

> In `TextTest.java` you can find a simple usecase on how to automatically test shadow DOM elements

Reviewer: ```suggestion
In `TextTest.java` you can find a simple use case on how to automatically test shadow DOM elements.
```

#### 1.2. Nested Shadow DOM

> In `CheckboxTest.java` there is are more sophisticated usecases for shadow DOM including nested shadow DOM's.

Reviewer: ```suggestion
`CheckboxTest.java` contains more sophisticated use cases for shadow DOM including nested shadow DOMs.
```

> In `CheckboxTest.java` there is are more sophisticated usecases for shadow DOM including nested shadow DOM's.

Reviewer: Also: If we have a wiki page for nested shadow DOMs or a section in our wiki about this, please place a hyperlink to the wiki page of the feature on "nested shadow DOMs."

## 2. Localization

> Under `showcase.neodymium.tests.localization` you can find how to use the localization feature of neodymium. Note that all the translated strings can be found in the `localization.yaml` in the config folder.

Reviewer: ```suggestion
The tests in the `showcase.neodymium.tests.localization` package highlight how the localization feature of neodymium is used. Note that all the translated strings and their keys can be found in the `config/localization.yaml` file.
```

> Under `showcase.neodymium.tests.localization` you can find how to use the localization feature of neodymium. Note that all the translated strings can be found in the `localization.yaml` in the config folder.

Reviewer: Also: Place a hyperlink to the wiki page of the feature on "localization feature."

# Content
Below is a summary of the content from this project.

## 1. Shadow DOM

> The `showcase.neodymium.tests.shadowdom` package contains the use cases for the shadow DOM automation. Please also have a look at our [wiki](https://github.com/Xceptance/neodymium-library/wiki/Shadow-DOM-Testing) for more information.

Reviewer: ```suggestion
The `showcase.neodymium.tests.shadowdom` package contains use cases for the shadow DOM automation. Please also have a look at our [wiki](https://github.com/Xceptance/neodymium-library/wiki/Shadow-DOM-Testing) for more information.
```

#### 1.1 General Shadow DOM
In `TextTest.java` you can find a simple use case on how to automatically test shadow DOM elements.
#### 1.2. Nested Shadow DOM
`CheckboxTest.java` contains more sophisticated use cases for shadow DOM including nested shadow DOMs. More information can be found [here](https://github.com/Xceptance/neodymium-library/wiki/Shadow-DOM-Testing).

## 2. Localization

> The tests in the `showcase.neodymium.tests.localization` package highlight use the [localization](https://github.com/Xceptance/neodymium-library/wiki/Localization) feature of neodymium is used. Note that all the translated strings and their keys can be found in the `config/localization.yaml` file. 

Reviewer: ```suggestion
The tests in the `showcase.neodymium.tests.localization` package highlight how the [localization](https://github.com/Xceptance/neodymium-library/wiki/Localization) feature of neodymium is used. Note that all the translated strings and their keys can be found in the `config/localization.yaml` file. 
```

> ## 3. Web slider 

Reviewer: If I'm seeing this correctly, this isn't part of the current `development` branch. Hence, this section should be removed and added when the web slider actually comes.

Author: As far as i know @jseng92 is working on this and added this section to the README. I would leave this in as this is basically in already.

Reviewer: It was added by him but we don't have a pull request for the slider yet. Hence, we would add something to the Readme, which isn't here yet and even if there was a pull request, we would introduce a state which does not have the slider-code but tells us in the Readme that their would be. Instead, what is added here, should rather be added to the pull request for the slider.

In other words, even if it is basically in already, it doesn't belong in this merge request and should be removed.

Author: Ok, i edited out.
